dragscroll.js

```diff
@@ -1,11 +1,16 @@
 /**
- * @fileoverview dragscroll - scroll area by dragging
- * @version 0.0.8
- * 
- * @license MIT, see http://github.com/asvd/dragscroll
- * @copyright 2015 asvd <heliosframework@gmail.com> 
+ * @fileoverview dragscroll-zoom - scroll area by dragging and zoom in~out
+ * @version 1.0.0
```

Comment:
I would use version 1.0.0 for a features that have been intensively tested and proved to be stable